### PR TITLE
O3-1221: Fetch active visits only when the location param is provided

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Summary
When fetching active visits with the `location` parameter as undefined, the backend throws the 404 error.  This PR enable fetching the active visits only when the `location` parameter is defined. 

## Screenshots

Screenshot 2022-05-19 at 5 13 00 PM

## Related Issue

*None.*

## Other

*None.*

### CODE REVIEW

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx [hunk 1]
```diff
@@ -31,7 +31,10 @@ export function useActiveVisits() {
     '&location=' +
     sessionLocation;
   const url = `/ws/rest/v1/visit?includeInactive=false&v=${customRepresentation}`;
-  const { data, error, isValidating } = useSWR<{ data: { results: Array<Visit> } }, Error>(url, openmrsFetch);
+  const { data, error, isValidating } = useSWR<{ data: { results: Array<Visit> } }, Error>(
+    sessionLocation ? url : '',
```
Comment (ibacher) [inline packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx:35]: `null` essentially tells SWR "don't run this request". `''` tells SWR "run the fetch with just the configured base url". See [the docs](https://swr.vercel.app/docs/conditional-fetching).

```suggestion
    sessionLocation ? url : null,
```

**Review by ibacher — COMMENTED** [review]
One change and it LGTM.

File: packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx (hunk 1 above)
Comment (jwnasambu) [reply packages/esm-active-visits-app/src/active-visits-widget/active-visits.resource.tsx:35]: @ibacher thanks so so much for the clear insight.

**Review by donaldkibet — APPROVED** [review]
LGTM Thanks @jwnasambu

**Pull request merged**